components/feature/downloads/src/main/res/layout/mozac_downloads_prompt.xml

```diff
@@ -0,0 +1,86 @@
+<RelativeLayout
```

Comment:
I think we should use ConstraintLayout and avoid RelativeLayout

Reply:
This was mostly pattern-matched from `mozac_site_permissions_prompt.xml` which uses a RelativeLayout. May I ask why it's important to use ConstraintLayout instead? 

If we change this one to that we should likely change both :)

Reply:
Yeah, it doesn't look like anything in AC uses ConstraintLayout, so I'd prefer to stick to RelativeLayout for now and do that as a holistic transition in the future :)